### PR TITLE
Articles and Comments are owned by the users that create them

### PR DESCRIPTION
**Pull request by santi698**
## Summary
* Added user to Article and Comment
* Users are not allowed to delete or modify other user's articles and comments

## Trello Card
https://trello.com/c/dhwtXtlg

## Screenshots
ss

### CODE REVIEW

File: db/migrate/20160122145339_add_user_ref_to_article.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToArticle < ActiveRecord::Migration
+  def change
+    add_reference :articles, :user, index: true, foreign_key: true
```
Comment (epintos) [inline db/migrate/20160122145339_add_user_ref_to_article.rb]: Be carefull that if you want to delete on cascade, the migration should have that

File: spec/controllers/article_controller_spec.rb [hunk 2]
```diff
@@ -147,6 +168,10 @@
         put :update, id: article, article: params
         expect(response).to render_template :edit
       end
+      it 'writes a flash notice' do
+        put :update, id: article, article: params
+        should set_flash[:error]
```
Comment (epintos) [inline spec/controllers/article_controller_spec.rb]: Better to use expect synatx

Comment (epintos) [inline spec/controllers/article_controller_spec.rb]: Rubocop should detect that

**Comment by epintos** [comment]
Few fixes and :+1:

**Comment by santi698** [comment]
Can I merge this one now?

**Comment by epintos** [comment]
:+1:

**Pull request merged**